.circleci/config.yml

```diff
@@ -52,89 +61,78 @@ commands:
-            ssh-keyscan -H ssh.$PLATFORM_REGION >> ~/.ssh/known_hosts
             ssh-keyscan -H github.com >> ~/.ssh/known_hosts
+            ssh-keyscan -H ssh.$PLATFORM_REGION >> ~/.ssh/known_hosts
+  install_psh_cli:
+    description: "Install the Platform.sh CLI tool"
+    steps:
       - run:
           name: Install the Platform.sh CLI tool
-          command: |
-            curl -sS https://platform.sh/cli/installer | php
-  github_hosts_workaround:
-    description: "Adds github.com to known hosts in container; for working locally."
-    steps:
+          command: curl -sS https://platform.sh/cli/installer | php
       - run:
-          name: Keyscan GitHub hostname
-          command: mkdir -p ~/.ssh && ssh-keyscan -H github.com >> ~/.ssh/known_hosts
+          name: Add platform cli tool to $PATH
+          command: echo 'export PATH="$HOME/"'.platformsh/bin':"$PATH"' >> $BASH_ENV
 jobs:
   # Tests the integrity of the build, stores the results in a workspace for re-use in later jobs.
   build:
     docker:
       - image: circleci/php:7.4.15-apache-browsers
+    environment:
+      PLATFORM_REGION: "uk-1.platform.sh"
     steps:
+      - hosts_keyscan
       - checkout_code
       - install_php_os_extensions
       - composer_tasks
       - persist_to_workspace:
           root: ./
           paths:
             - ./
-  build_edge_branch:
-    docker:
-      - image: circleci/php:7.4.15-apache-browsers
-    steps:
-      - checkout_code
-      - install_php_os_extensions
-      - run:
-          name: Switch to edge build branch
-          command: |
-            cd ~/project
-            git checkout -b $EDGE_BUILD_BRANCH
-      - run:
-          name: Install dev versions of dof-dss packages with composer.
-          command: |
-            composer require dof-dss/nidirect-site-modules:dev-development dof-dss/nidirect-migrations:dev-development \
-              dof-dss/nidirect-d8-test-install-profile:dev-development dof-dss/nicsdru_origins_theme:dev-development \
-              dof-dss/nicsdru_origins_modules:dev-development dof-dss/nicsdru_nidirect_theme:dev-development
-      - persist_to_workspace:
-          root: ./
-          paths:
-            - ./
-
-  # Test for coding standards.
+  # Test for coding standards - will inherit the workspace/filesystem changes from build step, above.
   coding_standards:
     docker:
       - image: circleci/php:7.4.15-apache-browsers
+    environment:
+      PROJECT_ROOT: "/home/circleci/project"
+      PLATFORM_REGION: "uk-1.platform.sh"
     steps:
       - attach_workspace:
           at: ./
       - run: sudo cp /home/circleci/project/.circleci/docker-php-circleci.ini /usr/local/etc/php/conf.d/
       - run:
           name: PHPCS analysis
-          command: /home/circleci/project/phpcs.sh /home/circleci/project "/home/circleci/project/web/modules/origins /home/circleci/project/web/modules/custom /home/circleci/project/web/modules/migrate /home/circleci/project/web/themes/custom"
```

Comment:
Used a bash variable to build up this list. It's much clearer to read and easier to maintain.